File: 1002.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main() {
	
	double raio, area, pi = 3.14159;;
	
	cin >> raio;
	
	area = pi*pow(raio, 2);
	
	cout << setprecision(4) << fixed;
	cout << "A=" << area << endl;
	
	return 0;
}

File: Go/1002.go
package main
 
import (
	"fmt"
	"math"
)

func main() {
	var n, r, area float64

	fmt.Scanf("%f", &r)

	n = 3.14159
	area = n * math.Pow(r, 2)

	fmt.Printf("A=%.4f\n", area)
}